### PR TITLE
[meteoblue] Add French translation

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by cweitkamp** [comment]
@ghys @clinique May I ask you for a quick review? Thanks.

**Comment by martinvw** [comment]
> I defer to those who know how how i18n works.

And I will defer to those that speak the language 😄

File: addons/binding/org.openhab.binding.meteoblue/ESH-INF/i18n/meteoblue_fr.properties [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+thing-type.config.meteoblue.weather.serviceType.label = Type de service
+thing-type.config.meteoblue.weather.serviceType.description = Type d'API � utiliser.
+thing-type.config.meteoblue.weather.serviceType.option.Commercial = Commercial
+thing-type.config.meteoblue.weather.serviceType.option.NonCommercial = Non Commercial
+
+thing-type.config.meteoblue.weather.location.label = Localisation
+thing-type.config.meteoblue.weather.location.description = Localisation en coordonn�es g�ographiques (latitude, longitude[, altitude]).
+
+thing-type.config.meteoblue.weather.timeZone.label = Fuseau horaire
+thing-type.config.meteoblue.weather.timeZone.description = Fuseau horaire de cette emplacement (Europe/Paris par exemple). Voir https://en.wikipedia.org/wiki/List_of_tz_database_time_zones .
```
Comment (ghys) [inline addons/binding/org.openhab.binding.meteoblue/ESH-INF/i18n/meteoblue_fr.properties]:
```suggestion
thing-type.config.meteoblue.weather.timeZone.description = Fuseau horaire de cet emplacement (Europe/Paris par exemple). Voir https://en.wikipedia.org/wiki/List_of_tz_database_time_zones .
```

Comment (lolodomo) [reply addons/binding/org.openhab.binding.meteoblue/ESH-INF/i18n/meteoblue_fr.properties]: Oups

**Pull request merged**